src/prisma/schema.prisma

```diff
@@ -2570,3 +2570,10 @@ model GitcoinProjectAttestation {
   schemaId       String
   attestationUID String
 }
+
+model ServiceWorkerSubscriptions {
```

Comment:
model should be singular: ServiceWorkerSubscription

Reply:
PushNotificationSubscription sounds good.

Reply:
I also added a source so we know what app is it for